.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -117,9 +117,9 @@ After you have the version increments, the updated checkpoints and the updated c
 
 - [ ] Make sure the PR with the new checkpoint hashes is already merged, or make it part of the changelog PR
 - [ ] Push the version increments and the updated changelog into a branch
-      (name suggestion, example: `v1.0.0-rc.X`)
+      (for example: `v1.0.0-rc.0-release` - this needs to be different to the tag name)
```

Comment:
```suggestion
      (for example: `publish-v1.0.0-rc.0` - this needs to be different to the tag name)
```

Reply:
I'd prefer to go with `bump`, because publishing the release happens after that branch merges.

```suggestion
      (for example: `bump-v1.0.0-rc.0` - this needs to be different to the tag name)
```